.github/workflows/win-release-candidate.yml

```diff
@@ -58,16 +58,11 @@ jobs:
     - name: Build Signed Release
       shell: bash
       run: |
-        f=$(mktemp)
-        echo -ne "${WINDOWS_SIGNING_PFX_BASE64}" > "${f}"
-        CERT=$(mktemp)
-        # We just need the name; certutil complains if the dest exists.
-        rm -f "${CERT}"
-        certutil -decode "${f}" "${CERT}"
```

Comment:
I've removed this because it's not needed any more. In fact, its presence was breaking things because the `env` config for this Step used to include `WINDOWS_SIGNING_PFX_BASE64` in the old build system but that's now been renamed to `CSC_LINK` so this had started failing.